src/components/BugLink.svelte

```diff
@@ -2,9 +2,15 @@
   import { getBugLinkTitle } from "../formatters/links";
 
   export let ref;
-  const url = ref.startsWith("http")
+  const url = ref.toString().startsWith("http")
     ? ref
     : `https://bugzilla.mozilla.org/show_bug.cgi?id=${ref}`;
 </script>
 
+<style>
+  a {
+    display: block;
+  }
+</style>
```

Comment:
👍 I noticed this yesterday just after landing, but didn't get around to filing an issue.